docs/en_US/Compression/QuantizationSpeedup.rst

```diff
@@ -0,0 +1,141 @@
+Speed up Mixed Precision Quantization Model
+===========================================
+
+*This feature is in Beta version.*
+
+Introduction
+------------
+
+Deep learning network has been computationly intensive and memory intensive 
+which increase the difficulty of deploying deep neural network model. Quantization is a 
+fundamental technology which is widely used to reduce memory footprint and speed up inference 
+process. Many frameworks begin to support quantization, but few of them support mixed precision 
+quantiation. Frameworks like haq only support simulated mixed precision quantization which will 
+not speed up the inference process. To get real speed up of mixed precision quantization and 
+help people get the real feedback from hardware, we design a tool which combine nni and tensorrt together.
```

Comment:
we design a tool which combine nni and tensorrt together -> we design a general framework with simple interface to allow NNI to connect different DL model optimization backends (e.g., TensorRT, NNFusion), which gives users an end-to-end experience that after quantizing their model with quantization algorithms, the quantized model can be directly speeded up with the connected optimization backend. NNI connects TensorRT at this stage, and will support more backends in the future.

Reply:
Added.